dips/dip-201.md

```diff
@@ -39,8 +39,11 @@ pub struct TwoChainTimeoutCertificate {
 }
 ```
 
-TwoChainTimeoutCertificate only carries a single quorum cert instead of 2f+1 as an optimization to save both bandwidth and verification cost.
-It keeps the TwoChainTimeout with the highest quorum cert and the signature is signed on the derived struct
+The signature for TwoChainTimeout is signed on the derived struct TimeoutSigningRepr which uses the highest qc round instead of the actual QuorumCert 
```

Comment:
```suggestion
The signature for TwoChainTimeout is done **indirectly**, on the derived struct TimeoutSigningRepr which uses the highest qc round instead of an actual QuorumCert 
```